homeassistant/components/risco/sensor.py

```diff
@@ -91,7 +92,7 @@ def native_value(self):
         if self._event is None:
             return None
 
-        return self._event.time
+        return dt_util.as_local(dt_util.parse_datetime(self._event.time))
```

Comment:
Actually, I think this as_local may not be necessary as the state classes may notice this and adjust any timezones as needed. Do you know for sure this is required?

Reply:
I'm thinking about the code here that's adjusting timezones https://github.com/home-assistant/core/blob/c65a50bd2e40af76b01669c9d38c3916df59f0bd/homeassistant/components/sensor/__init__.py#L422

Reply:
On my local development environment it didn't work without `as_local`, and the error was about the sensor missing time zone info. I can reproduce this tomorrow to get the exact error if it helps, but I think `as_local` is needed in this case.

Reply:
So the string is parsed w/o a timezone, then added as local, then converted to UTC which all makes sense in the parent class. Thanks for talking me through this.

Reply:
```suggestion
        return dt_util.parse_datetime(self._event.time).astimezone(dt_util.UTC)
```